app/controllers/bids_controller.rb

```diff
@@ -17,16 +17,13 @@ def create
 
     @bid.save
     if @bid.save
-       redirect_to auction_item_path(@auction, @item), notice: "Thank You"
+      redirect_to auction_item_path(@auction, @item), notice: "Thank You"
     else
-      redirect_to auction_item_path(@auction, @item), notice: "Please revise your bid"
+      redirect_to auction_item_path(@auction, @item), notice: "Please revise your bid #{@bid.errors.full_messages}"
```

Comment:
Do you like how this is formatting? This is the most basic way to get messages in and it might not look that great. Think about maybe formatting this better than we did when debugging.